src/ParkrunMap.UI/ParkrunMap.UI/ClientApp/src/components/ParkrunMap.js

```diff
@@ -33,10 +34,34 @@ function mapDispatchToProps(dispatch) {
 }
 
 function mapStateToProps(state) {
-  return {
+
+  const props = {
     location: state.location.location,
     parkruns: state.parkruns.parkruns
+  };
+
```

Comment:
This much logic in the connect function scares me. Personally I would co-locate this with the state as a separate function (we usually refer to these as "selector" functions. The function would accept the parts of state that were relevant as params and return the relevant state.